places/pmorto-sanramon.md

```diff
@@ -0,0 +1,15 @@
+# San Ramon, CA
+
+### Places to Eat
+- Thai House
+- Four Flavors
+- The Brass Door
```

Comment:
Thai House is the best Thai restaurant in San Ramon and Danville. Four Flavors is interesting... it's like a Chipotle for Asian Fusion food. Really Good.